packages/clerk-js/src/ui/signIn/SignInFactorOne.tsx

```diff
@@ -38,7 +37,7 @@ function _SignInFactorOne(): JSX.Element {
     return <LoadingScreen />;
   }
 
-  if (showAllStrategies) {
+  if (showAllStrategies || !isSignInFactorOne(currentFactor.strategy)) {
```

Comment:
🔧 Since `showAllStrategies` is already in place, I'd suggest to enhance the [initial state logic](https://github.com/clerkinc/javascript/blob/276ef8e7a332f8604ef9956af6386db14187a68a/packages/clerk-js/src/ui/signIn/SignInFactorOne.tsx#L20) and determine whether it should be true by default when only OAuth first factors are used.